resources/definitions/fdmprinter.def.json

```diff
@@ -3529,6 +3529,19 @@
                     "enabled": "retraction_hop_enabled and extruders_enabled_count > 1",
                     "settable_per_mesh": false,
                     "settable_per_extruder": true
+                },
+                "retraction_hop_after_extruder_switch_height": 
```

Comment:
Just like the last PR, it would be good to add this too to one of the cfg-files that regulate when to show the setting, like `expert.cfg`.

Reply:
Thanks, retraction_hop_after_extruder_switch_height was added to expert.cfg as you suggested